README.md

```diff
@@ -29,18 +29,18 @@ This S3 Bucket (`s3://noaa-nws-owp-fim`) is set up as a "Requester Pays" bucket.
 
 #### Examples
 
-List bucket folder structure:
+The available versions can be found by running
 ```
-aws s3 ls s3://noaa-nws-owp-fim/ --request-payer requester
+aws s3 ls s3://noaa-nws-owp-fim/hand_fim/ --request-payer requester
 ```
 
-Download a directory of outputs for a HUC8:
+Example: download a directory of outputs for a single HUC8:
```

Comment:
Small potatoes, but there is already an Examples subheading on line 30, so having `Example` here is redundant.